.circleci/config.yml

```diff
@@ -108,298 +95,82 @@ jobs:
-      - run:
-          name: 'Test transaction-manager'
-          command: 'yarn workspace @requestnetwork/transaction-manager run test --ci --maxWorkers=1'
+          path: packages/request-node/reports/
       - store_test_results:
           path: packages/transaction-manager/reports/
-  test-ethereum-storage:
-    docker:
-      - *node_image
-      - *ipfs_image
-      - *ganache_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Build ethereum-storage'
-          command: 'yarn workspace @requestnetwork/ethereum-storage run build'
-      - run:
-          name: 'Deploy contracs smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
-      - run:
-          name: 'Test ethereum-storage'
-          command: 'yarn workspace @requestnetwork/ethereum-storage run test --ci --maxWorkers=1'
       - store_test_results:
-          path: packages/ethereum-storage/reports/
-  test-smart-contracts:
-    docker:
-      - *node_image
-      - *ganache_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Deploy contracs smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run deploy'
-      - run:
-          name: 'Lint smart contracts smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run lint:sol:check'
-      - run:
-          name: 'Test smart contracts smart-contracts'
-          command: 'yarn workspace @requestnetwork/smart-contracts run test'
-      - run:
-          name: 'Test smart contracts utilities'
-          command: 'yarn workspace @requestnetwork/smart-contracts run test:lib --ci --maxWorkers=1'
-  test-request-logic:
-    docker:
-      - *node_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Build request-logic'
-          command: 'yarn workspace @requestnetwork/request-logic run build'
-      - run:
-          name: 'Test request-logic'
-          command: 'yarn workspace @requestnetwork/request-logic run test --ci --maxWorkers=1'
+          path: packages/epk-signature/reports/
       - store_test_results:
-          path: packages/request-logic/reports/
-  test-advanced-logic:
-    docker:
-      - *node_image
-    working_directory: *working_directory
-    steps:
-      - attach_workspace:
-          at: *working_directory
-      - run:
-          name: 'Build advanced-logic'
-          command: 'yarn workspace @requestnetwork/advanced-logic run build'
-      - run:
-          name: 'Test advanced-logic'
-          command: 'yarn workspace @requestnetwork/advanced-logic run test --ci --maxWorkers=1'
+          path: packages/web3-signature/reports/
```

Comment:
I think there is an issue with this, see the [last workflow run](https://app.circleci.com/pipelines/github/RequestNetwork/requestNetwork/10347/workflows/da567f79-15c7-449c-a5ff-892d8b49c756/jobs/159253). 
> Unable to save test results from /home/circleci/repo/packages/web3-signature/reports
Error error accessing path "/home/circleci/repo/packages/web3-signature/reports": stat /home/circleci/repo/packages/web3-signature/reports: no such file or directory
Found no test results, skipping

It looks like we don't have any report in the "Tests" tab:
https://app.circleci.com/pipelines/github/RequestNetwork/requestNetwork/10347/workflows/da567f79-15c7-449c-a5ff-892d8b49c756/jobs/159253/tests

Reply:
I will look into this one

Reply:
I investigated further and running the eslint from the root of the project doesn't have errors.

Here is an explanation why that could happen

<img width="611" alt="Screenshot 2024-12-09 at 18 03 59" src="https://github.com/user-attachments/assets/096b70f7-10db-4045-af52-12aacccc7f49">